### PR TITLE
Enum for ChatMember statuses and Message#hasCaption

### PR DESCRIPTION
**Pull request by nordiauwu**

### CODE REVIEW

File: telegrambots-meta/src/main/java/org/telegram/telegrambots/meta/api/objects/ChatMember.java [hunk 1]
```diff
@@ -189,4 +189,13 @@ public String toString() {
                 ", customTitle=" + customTitle +
                 '}';
     }
+
+    public enum Status {
```
Comment (rubenlagus) [inline telegrambots-meta/src/main/java/org/telegram/telegrambots/meta/api/objects/ChatMember.java:193]: Please move this to a separate file

Comment (nordiakt) [reply telegrambots-meta/src/main/java/org/telegram/telegrambots/meta/api/objects/ChatMember.java]: +

**Review by rubenlagus — APPROVED** [review]

**Comment by rubenlagus** [comment]
Happy to merge this change but please extract the status class to a different class and name it with something that indicate it refers to chat member statuses.

**Comment by nordiauwu** [comment]
Outdated

**Pull request closed without merging**